docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '3'
+services:
+  
+  # # Frontend Service
+  frontend:
+    image: jasminegraph-frontend:latest  # Replace with your frontend image
+    ports:
+      - "3000:3000"
+    networks:
+      - next-node
+    environment:
+      - BACKEND=http://backend:8080 # Pointing to the backend service
+      - NEXT_PUBLIC_API_URL=http://backend:8080 # Pointing to the backend service
+    depends_on:
+      - backend  # Ensure backend service starts before frontend
+
+  # Backend Service
+  backend:
+    image: jasminegraph-backend:latest  # Replace with your backend image
+    ports:
+      - "8080:8080"
+    networks:
+      - node-mongo
+      - next-node
+    depends_on:
+      - mongoCont  # Ensure MongoDB starts before backend
+    environment:
+      - MONGO_URL=mongodb://mongoCont:27017/jasmine # MongoDB connection string pointing to the mongo service
+      - SERVER_PORT=7776
+      - SERVER_HOST=10.8.100.245
```

Comment:
Why is this specific IP address?

Reply:
these are use for testing purpost and i removed them